src/devices/PiJuice/Models/RSOCEstimationType.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.PiJuiceDevice.Models
+{
+    /// <summary>
+    /// Battery relative state-of-health
+    /// </summary>
+    public enum RSOCEstimationType
```

Comment:
RsocEstimationType or a long name rather than Rsoc. It's not ver know abbreviation